src/components/About/index.js

```diff
@@ -0,0 +1,27 @@
+import React, { useState } from 'react';
+import "./about.scss";
+import Structure from '../../structure.json';
+import aboutImage1 from '../../assets/img/about_img_1.png';
+import aboutImage2 from '../../assets/img/about_img_2.png';
+
+
+const About = () => {
+    const [about] = useState(Structure['about']);
+    const {title, subTitle, text} = about;
```

Comment:
You don't have declare variables because you can access to data using `about.title/about.subTitle/about.text`